Cargo.toml

```diff
@@ -1,8 +1,10 @@
 [package]
 name = "mongodb"
 version = "0.1.0"
-authors = ["Valeri Karpov <valkar207@gmail.com>"]
+authors = ["Valeri Karpov <valkar207@gmail.com>",
+           "Saghm Rossi <saghmrossi@gmail.com",
+           "Kevin Yeh <kevinyeah@utexas.edu"]
 
 [dependencies]
-
-libc="0.1.8"
+byteorder = "*"
```

Comment:
Not sure about rust, but using my node-based knee-jerk reaction is 'dont use `*` ever'. In rust '~0.3' and '~0.1' should be sufficient because of cargo.lock. Also, small nitpick, but it helps to keep deps in alphabetical order.